src/content/blog/astro-4120.mdx

```diff
@@ -0,0 +1,140 @@
+
+| <span class="whitespace-nowrap">Metrics</span>                                                                                                 | Partial Prerendering                                                 | Server Islands                                                       |
+| ---------------------------------------------------------------------------------------------------------------------------------------------- | -------------------------------------------------------------------- | -------------------------------------------------------------------- |
+| <abbr class="sm:hidden no-underline" title="Time To First Byte">TTFB</abbr><span class="hidden sm:inline">Time To First Byte</span>            | 0.838s                                                               | 0.766s                                                               |
+| <abbr class="sm:hidden no-underline" title="First Contentful Paint">FCP</abbr><span class="hidden sm:inline">First Contentful Paint</span>     | 1.740s                                                               | 1.251s                                                               |
+| <abbr class="sm:hidden no-underline" title="Largest Contentful Paint">LCP</abbr><span class="hidden sm:inline">Largest Contentful Paint</span> | 1.740s                                                               | 1.405s                                                               |
+| <abbr class="sm:hidden no-underline" title="Total Blocking Time">TBT</abbr><span class="hidden sm:inline">Total Blocking Time</span>           | 0.093s                                                               | 0.018s                                                               |
+| Source                                                                                                                                         | [WebPageTest](https://www.webpagetest.org/result/240715_BiDc15_G7W/) | [WebPageTest](https://www.webpagetest.org/result/240716_AiDc6D_D1N/) |
+
+As you can see from this table, the numbers are extremely close. Server Islands has an advantage in painting slightly faster due to being more lightweight both in the infrastructure setting in front of the request, and the amount of code needed in the client throughout the process. Partial prerendering has an advantage in that it can finish slightly faster, due to starting the request for its deferred content on the server before the HTML reaches the user's browser.
+
+Overall we are happy with the trade-offs we've made for Server Islands. Our implementation happens mostly at build-time, where component content is swapped out for a small script, one that can almost fit into a Tweet. This means that Server Islands aren't dependant on any particular server infrastructure and will run on any host.
```

Comment:
"This means that Server Islands aren't dependant on any particular server infrastructure and will run on any host." 

This is SUPER important, but added here at the end almost as a throw away / an accidental side-effect of our approach. Can the portability angle get its own paragraph?